docs/docs/adding-analytics.md

```diff
@@ -37,16 +37,20 @@ npm install --save gatsby-plugin-google-analytics
 ```
 
 ```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
     {
-      resolve: `gatsby-plugin-google-analytics`,
+      resolve: 'gatsby-plugin-google-analytics',
       options: {
-      // replace "UA-XXXXXXXXX-X" with your own Tracking ID
+        // replace "UA-XXXXXXXXX-X" with your own Tracking ID
         trackingId: "UA-XXXXXXXXX-X",
       },
     },
+  ],
+};
```

Comment:
If you remove this semicolon, the linting will pass.

Reply:
fixed